cypress/pageobject/Users/UserCreation.ts

```diff
@@ -1,65 +1,106 @@
 // UserCreation.ts
 export class UserCreationPage {
-  clickElementById(elementId: string) {
-    cy.get("#" + elementId).click();
+  clickProfileName() {
+    cy.get("#user-profile-name").click();
   }
-
-  typeIntoElementById(elementId: string, value: string) {
-    cy.get("#" + elementId)
-      .click()
-      .type(value);
+  clickProfileButton() {
+    cy.verifyAndClickElement("#profile-button", "Profile");
   }
-
-  typeIntoElementByIdPostClear(elementId: string, value: string) {
-    cy.get("#" + elementId)
-      .click()
-      .clear()
-      .click()
-      .type(value);
+  clickEditProfileButton() {
```

Comment:
```suggestion
  clickEditProfileButton() {
```

- we already have a clickEditProfileButton() function , dont duplicate the function, use the one in the userProfilePage.ts

Reply:
Used